packages/yourloops/lib/auth/hook.tsx

```diff
@@ -506,6 +507,17 @@ export function AuthContextImpl(api: AuthAPI): AuthContext {
     // return unmount;
   };
 
+  const certifyProfessionalAccount = async (): Promise<void> => {
+    if (!user) {
+      throw Error("User not logged in");
+    }
+    const { frProId } = await api.certifyProfessionalAccount();
+    user.frProId = frProId;
+    setUser(user);
+  };
+
+  const redirectToProfessionalAccountLogin = (): void => window.location.assign(`${appConfig.API_HOST}/auth/oauth/login`);
```

Comment:
You should use the history hook like everywhere else

Reply:
We can't redirect to an external url with React Router